release-notes.md

```diff
@@ -52,8 +68,40 @@ This release has the following security issues:
 - Fixed a potential race condition during reconciliation of ResourceClaims which might cause the Services Toolkit manager to stop responding.
 - Updated configuration of the Services Toolkit carvel Package to prevent an unwanted build up of ConfigMap resources.
 
+#### Supply Chain Security Tools - Store
+
+- Fixed an issue where querying a source report with local path name would return the following error: `{ "message": "Not found" }`
+
 ### <a id='1-1-known-issues'></a> Known issues
 
+#### Supply Chain Security Tools - Store
+
+- **Persistent volume retains data**
+  If Supply Chain Security Tools - Store is deployed, deleted, and then redeployed the
+  `metadata-store-db` Pod fails to start if the database password changed during redeployment.
+  This is caused by the persistent volume used by postgres retaining old data, even though the retention
+  policy is set to `DELETE`.
+
+  To resolve this issue, see [CrashLoopBackOff from Password Authentication Fails](troubleshooting.html#password-authentication-fails)
+  in _Troubleshooting Tanzu Application Platform_.
+
+  > **Warning:** Changing the database password deletes your Supply Chain Security Tools - Store data.
+
+- **Missing persistent volume:**
+  After Supply Chain Security Tools - Store is deployed, `metadata-store-db` Pod might fail for missing
+  volume while `postgres-db-pv-claim` pvc is in the `PENDING` state.
+  This issue may occur if the cluster where Supply Chain Security Tools - Store is deployed does not have
+  `storageclass` defined.
+
+  The provisioner of `storageclass` is responsible for creating the persistent volume after
+  `metadata-store-db` attaches `postgres-db-pv-claim`. To resolve this issue, see
+  [Missing Persistent Volume](troubleshooting.html#missing-persistent-volume)
+  in _Troubleshooting Tanzu Application Platform_.
```

Comment:
I'm assuming this was copied from the somewhere else, but the newlines in this whole known issues section is a little strange... Was that intentional or a bug?

Reply:
Despite the new lines here, in the markdown output view it's single line. This was edited by docs team. I think they did this to not have these line too long.